README.md

```diff
@@ -14,7 +14,11 @@
 -->
 # ⚠️ Archive Notice
 
-As of October 24, 2023, we will no longer update or support the [Single-Instance OSS/Pro Kubernetes Chart](https://github.com/sonatype/nxrm3-helm-repository/tree/main/nexus-repository-manager).
+As of October 24, 2023, we will no longer update or support the [Single-Instance OSS/Pro Helm Chart](https://github.com/sonatype/nxrm3-helm-repository/tree/main/nexus-repository-manager). 
+
+Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. We strongly recommend that you use an external PostgreSQL database for Kubernetes deployments. If you are deploying in AWS, you can use our [AWS Helm chart](https://github.com/sonatype/nxrm3-helm-repository/tree/main/nxrm-aws-resiliency) to deploy Nexus Repository in an EKS cluster.
+
+We do not currently provide Helm charts for on-premises deployments using PostgreSQL. Please see our [Installation and Upgrades Help Documentation](https://help.sonatype.com/repomanager3/installation-and-upgrades).
```

Comment:
Minor: I feel like a why is missing in the last sentence.  e.g. for more info (probably that's not why).

Reply:
@joedragons A why for what specifically? This was the "why" we were adding: "Deploying Nexus Repository in containers with an embedded database has been known to corrupt the database under some circumstances. " to cover why we are deprecating...

Reply:
oh you mean a why for why we don't have on-prem helm charts? @vswaminathan777 would have to comment on that. We just only have the AWS one at this point.

Reply:
Sorry no the last sentence is "Please see our docs".

Reply:
@joedragons We do not have a Helm chart for on-prem that can work with PostgreSQL DB but I think you have made a valid point here.
@lisadurant Instead of the Installation docs link, I think we should be re-directing users to on-prem resiliency YAMLs and docs (this is PostgreSQL based) to help users to setup NXRM in a Kubernetes cluster.

Reply:
@vswaminathan777 @joedragons realized I was addressing the wrong issue - I was providing a pointer to basically the manual method you'd have to use if you continued using an embedded database, forgetting I was literally talking about PostgreSQL. I've revised now.